.env

```diff
@@ -1,9 +1,14 @@
-DUCOR_EOS_ORACLE_PRIVATEKEY=5HqECDpMfwJcdsUVKgkGQHXy8XBaubqxUnyDcazP9TXvuXQVatx
-DUCOR_EOS_ORACLE_ACCOUNT=workshop2221
+DEBUG=ERR,WARNING,INFO
```

Comment:
Can we prefix this var? Like `LOG_LEVEL` or something.

Reply:
Yes we can!